analyzers/tests/SonarAnalyzer.UnitTest/TestCases/MethodParameterUnused.RoslynCfg.cs

```diff
@@ -105,7 +106,8 @@ void LocalFunctionLocalLifetime(int localArgLocalLifetime)
                 }
             }
 
-            void DoSomething() { }
+            void DoSomething()
+            { }
```

Comment:
Any reason for this change or just automatic formating?

Reply:
VS fixes some formatting errors when you copy/paste the expected result. I changed the code, so it is now copy/paste friendly. (I fixed the "elastic marker" problem for the same reason).